sklearn/linear_model/tests/test_omp.py

```diff
@@ -88,15 +87,15 @@ def test_unreachable_accuracy():
                           n_nonzero_coefs=n_features))
 
 
-def test_bad_input():
-    assert_raises(ValueError, orthogonal_mp, X, y, tol=-1)
-    assert_raises(ValueError, orthogonal_mp, X, y, n_nonzero_coefs=-1)
-    assert_raises(ValueError, orthogonal_mp, X, y,
-                  n_nonzero_coefs=n_features + 1)
-    assert_raises(ValueError, orthogonal_mp_gram, G, Xy, tol=-1)
-    assert_raises(ValueError, orthogonal_mp_gram, G, Xy, n_nonzero_coefs=-1)
-    assert_raises(ValueError, orthogonal_mp_gram, G, Xy,
-                  n_nonzero_coefs=n_features + 1)
+# Parametrizing the decorator to enhance the test
```

Comment:
This comment would apply for any use of `@pytest.mark.parametrize` and is therefore not specific enough to be useful. Let's just remove it.

```suggestion
```

Reply:
The reason I put these small changes between commits is that I update my merge conflicts (the main objective) but I can't commit to show that my changes are now good to go. Do you have any recommended resources to read to get better in git?